src/gui/advancedsettings.cpp

```diff
@@ -523,6 +529,15 @@ void AdvancedSettings::loadAdvancedSettings()
     m_spinBoxDiskQueueSize.setSuffix(tr(" KiB"));
     addRow(DISK_QUEUE_SIZE, (tr("Disk queue size") + u' ' + makeLink(u"https://www.libtorrent.org/reference-Settings.html#max_queued_disk_bytes", u"(?)"))
             , &m_spinBoxDiskQueueSize);
+#ifdef QBT_USES_LIBTORRENT2
+    // Disk IO type
+    m_comboBoxDiskIOType.addItem(tr("Default"), QVariant::fromValue(BitTorrent::DiskIOType::Default));
+    m_comboBoxDiskIOType.addItem(tr("Memory mapped files"), QVariant::fromValue(BitTorrent::DiskIOType::MMap));
```

Comment:
>tr("Memory mapped files")

Now gui and webui has different options displayed, please unify them.

Reply:
> > tr("Memory mapped files")
> 
> Now gui and webui has different options displayed, please unify them.

I just made it the way I would like it to look in the GUI. If WebUI goes the other way, I don't care. I don't mind if it is unified. But I don't like how it looks in WebUI now, so I used more user friendly option names in GUI.

Reply:
Well the collaborators of this PR should sort it out... I think the webui version is nicer, perhaps GUI should follow it.

Off topic, this is why I prefer to submit a supersede PR instead of joining one...

Reply:
Okay, whatever. Continue as you wish (but taking into account the requirements for the correct spelling of abbreviations).

It just wouldn't look very acceptable to me if it was proposed to merge with a missing GUI. I hope it will not be difficult for the @mayli to make the requested changes in GUI part. Otherwise you can drop my commit.

Reply:
oops, I corrected my [previous post](https://github.com/qbittorrent/qBittorrent/pull/16895#discussion_r856846071).

Reply:
reworded the webui options.